### PR TITLE
[hi] Add content/hi/docs/contribute/analytics.md

### PR DESCRIPTION
**Pull request by Shubh28698**
Hello!

This PR is a part of ongoing hi localization effort

Hindi localization of page: content/en/docs/contribute/analytics.md

/language hi

@anubha-v-ardhan  Can you please review

### CODE REVIEW

**Comment by Shubh28698** [comment]
I signed it

**Comment by Shubh28698** [comment]
I signed it

**Comment by Shubh28698** [comment]
> /check-cla

I signed it and even received the reply for succesfully signing

**Comment by Shubh28698** [comment]
I signed it

File: content/hi/docs/contribute/analytics.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+[डैशबोर्ड देखें](https://datastudio.google.com/reporting/fede2672-b2fd-402a-91d2-7473bdb10f04)।
+
+यह डैशबोर्ड Google Data Studio का उपयोग करके बनाया गया है और Google Analytics का उपयोग करके Kubernetes.io पर एकत्रित जानकारी दिखाता है।
+
+### डैशबोर्ड का उपयोग करना
+
+डिफ़ॉल्ट रूप से, डैशबोर्ड पिछले 30 दिनों के सभी एकत्रित विश्लेषण दिखाता है। विभिन्न दिनांक सीमा मे आने वाला डेटा देखने के लिए date selector का उपयोग करें। अन्य फ़िल्टरिंग विकल्प आपको उपयोगकर्ता का स्थान, साइट तक पहुंचने के लिए उपयोग किए जाने वाले उपकरण, उपयोग किए गए दस्तावेज़ों के अनुवाद, और बहुत से चिज़ो का डेटा देखने की अनुमति देते हैं।
+
+ यदि आप इस डैशबोर्ड में कोई समस्या देखते हैं, या किसी सुधार का अनुरोध करना चाहते हैं, तो कृपया [एक समस्या खोलें](https://github.com/kubernetes/website/issues/new/choose)।
```
Comment (rajatgupta24) [inline content/hi/docs/contribute/analytics.md:25]: I think this is more understandable
```suggestion
 यदि आप इस डैशबोर्ड में कोई समस्या देखते हैं, या किसी सुधार का अनुरोध करना चाहते हैं, तो कृपया [एक इशू बनाएं](https://github.com/kubernetes/website/issues/new/choose)।
```

**Review by rajatgupta24 — CHANGES_REQUESTED** [review]

**Comment by anubha-v-ardhan** [comment]
/hold
while we welcome and value your effort to contribute to HIndi localization, we will need quite some time to accept it as our first priority is to launch the localization on k8s.io. Once thats done, our reviewers will be available to review this PR but we can still have this on hold :)

File: content/hi/docs/contribute/analytics.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+<!-- body -->
+
+[डैशबोर्ड देखें](https://datastudio.google.com/reporting/fede2672-b2fd-402a-91d2-7473bdb10f04)।
+
+यह डैशबोर्ड Google Data Studio का उपयोग करके बनाया गया है और Google Analytics का उपयोग करके Kubernetes.io पर एकत्रित जानकारी दिखाता है।
+
+### डैशबोर्ड का उपयोग करना
+
+डिफ़ॉल्ट रूप से, डैशबोर्ड पिछले 30 दिनों के सभी एकत्रित विश्लेषण दिखाता है। विभिन्न दिनांक सीमा मे आने वाला डेटा देखने के लिए date selector का उपयोग करें। अन्य फ़िल्टरिंग विकल्प आपको उपयोगकर्ता का स्थान, साइट तक पहुंचने के लिए उपयोग किए जाने वाले उपकरण, उपयोग किए गए दस्तावेज़ों के अनुवाद, और बहुत से चिज़ो का डेटा देखने की अनुमति देते हैं।
```
Comment (jayesh-srivastava) [inline content/hi/docs/contribute/analytics.md:23]: It should be `चीज़ों` instead of `चिज़ो`.

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]

File: content/hi/docs/contribute/analytics.md (hunk 2 above)
Comment (Shubh28698) [reply content/hi/docs/contribute/analytics.md]: Its done. Thanks for the review

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Review by anubha-v-ardhan — APPROVED** [review]
/approve
Thank you @Shubh28698 :)

**Pull request merged**